lib/grandpa/message.go

```diff
@@ -77,21 +77,30 @@ func (v *VoteMessage) ToConsensusMessage() (*ConsensusMessage, error) {
 	}, nil
 }
 
-// NeighbourMessage represents a network-level neighbour message
-type NeighbourMessage struct {
-	Version byte
-	Round   uint64
-	SetID   uint64
-	Number  uint32
+type VersionedNeighborMessage scale.VaryingDataType
```

Comment:
Should we just keep it as `NeighbourMessage` though? I don't think there is a need to rename it to `VersionedNeighborMessage` right?

Reply:
from what I understand we could support more than one neighbour messages but for now, only the version 1 is in usage. I believe we can keep it as it is